FE/components/contributors/index.tsx

```diff
@@ -0,0 +1,33 @@
+import Image from 'next/image';
+import { Fragment } from 'react';
+import { IContributors } from '../../interfaces';
+
+type Props = {
+  contributors: IContributors[];
+};
+
+const ContributorList = ({ contributors }: Props) => {
+  return (
+    <Fragment>
+      {contributors.map((contributor, index) => (
+        <a
+          key={index}
+          href={contributor.html_url}
+          target="_blank"
+          rel="noreferrer"
+          title={contributor.html_url.split('/')[3]}
+        >
+          <Image
+            src={contributor.avatar_url}
+            alt={contributor.html_url.split('/')[3]}
+            width={30}
+            height={30}
+            tw="rounded-full bg-black"
```

Comment:
you've not imported tw from 'twin.macro' in this component. that's why images are squared. please add an import statement for tw.

Reply:
There you go, it's added.